generators/languages/index.js

```diff
@@ -97,16 +105,14 @@ module.exports = class extends BaseBlueprintGenerator {
             },
 
             validate() {
-                this.currentLanguages = this.jhipsterConfig.languages || [];
-                if (this.languages) {
+                if (this.languagesToApply) {
                     if (this.skipClient) {
-                        this.log(chalk.bold(`\nInstalling languages: ${this.languages.join(', ')} for server`));
+                        this.log(chalk.bold(`\nInstalling languages: ${this.languagesToApply.join(', ')} for server`));
                     } else if (this.skipServer) {
-                        this.log(chalk.bold(`\nInstalling languages: ${this.languages.join(', ')} for client`));
+                        this.log(chalk.bold(`\nInstalling languages: ${this.languagesToApply.join(', ')} for client`));
                     } else {
-                        this.log(chalk.bold(`\nInstalling languages: ${this.languages.join(', ')}`));
+                        this.log(chalk.bold(`\nInstalling languages: ${this.languagesToApply.join(', ')}`));
                     }
-                    this.languagesToApply = this.languages || [];
                 } else {
                     this.log(chalk.bold('\nLanguages configuration is starting'));
```

Comment:
@mshima : should we remove this line ? As it appears when anwsering to the questions:

```
? Which *type* of application would you like to create? Monolithic application (recommended for simple projects)
? [Beta] Do you want to make it reactive with Spring WebFlux? No
? What is the base name of your application? burger

Languages configuration is starting <------------------------
? What is your default Java package name? io.github.pascalgrimaud.burger
? Do you want to use the JHipster Registry to configure, monitor and scale your application? No
? Which *type* of authentication would you like to use? JWT authentication (stateless, with a token)
? Which *type* of database would you like to use? SQL (H2, MySQL, MariaDB, PostgreSQL, Oracle, MSSQL)
? Which *production* database would you like to use? MySQL
? Which *development* database would you like to use? H2 with disk-based persistence
? Do you want to use the Spring cache abstraction? Yes, with the Ehcache implementation (local cache, for a single node)
? Do you want to use Hibernate 2nd level cache? Yes
? Would you like to use Maven or Gradle for building the backend? Maven
? Which other technologies would you like to use? 
? Which *Framework* would you like to use for the client? 
  Angular 
  React 
❯ Vue 
  No client 
```